internal/services/housekeeper/tablePartitionCleaner.go

```diff
@@ -17,13 +17,13 @@ func DeletePartitions() {
 	retentionThresholdDate := currentTime.AddDate(0, 0, -cfg.DataRetentionPeriod)
```

Comment:
Any thoughts on RDS time `us-east-1` or standard utc for currentTime instead of time.Now()?

Reply:
In the cost report we get timestamp in UTC, so make sense to convert this too in UTC.